string[]    observed_objects
float32[]   objects_relative_x
float32[]   objects_relative_y
bool[]      objects_grasped
